### PR TITLE
Iriska

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Very fat and lazy cat for the captain.

Iriska too lazy to leave her pillow, and will bite those who she doesn't like, meaning everyone but captain and other pets.
Can be bribed with food. Not eating bananas and not accepting food from hands.
Doesn't like to be hurt, will never forgive you, and if she dies - curse will fall upon those who attacked her.
Maneki Neko kind of curse, so people don't just kill Iriska casually.

image

Planned feature - Iriska likes her pillow pulled around non-maintance areas of the ship.
Would be nice to reward players for doing so, but don't know what is acceptable.
Temp stats? Extra luck in junking? ~~Hairball oddity?~~ Something different?

Also fixed bones hissing in dead chat.

## Why It's Good For The Game
You know why.
## Changelog
:cl:
add: Iriska the cat
fix: Bones should no longer hiss in dead chat
imageadd: Iriska sprites by Bloodpriest
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/iriska.dm [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+	health = 80
+	maxHealth = 80
+	wander = FALSE
+	canmove = FALSE
+	speak_emote = list("purrs.", "meows.")
+	emote_see = list("shakes her head.", "shivers.")
+	speak_chance = 0.75
+	meat_amount = 6
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
+	response_help  = "pets"
```
Comment (Firefox13) [inline code/modules/mob/living/simple_animal/friendly/iriska.dm:15]:
```suggestion
	response_help = "pets"
```

File: code/modules/mob/living/simple_animal/friendly/iriska.dm [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+	response_disarm = "rubs"
+	response_harm   = "makes terrible mistake by kicking"
```
Comment (Firefox13) [inline code/modules/mob/living/simple_animal/friendly/iriska.dm:17]:
```suggestion
	response_harm = "makes terrible mistake by kicking"
```

File: code/modules/mob/living/simple_animal/friendly/iriska.dm [hunk 3]
```diff
@@ -0,0 +1,203 @@
+/mob/living/simple_animal/iriska
+	name = "Iriska"
+	desc = "The captain's own cat. Fat and lazy."
```
Comment (drexample) [inline code/modules/mob/living/simple_animal/friendly/iriska.dm:3]: It's the other way around. Captain is Iriska's pet

**Comment by drexample** [comment]
Btw, there are pillowless iriska sprites too.

File: code/modules/mob/living/simple_animal/friendly/iriska.dm (hunk 3 above)
Comment (SirRichardFrancis) [reply code/modules/mob/living/simple_animal/friendly/iriska.dm:3]: I agree. Initially there was something like "An actual captain of the ship".

**Comment by SirRichardFrancis** [comment]
> Btw, there are pillowless iriska sprites too.

Want pillow to be removable? Think an individual with high enough ROB and spare life can do that.

**Comment by SirRichardFrancis** [comment]
This branch was split from master right before PR with 1,253 files changed. Will try to transfer those to this branch, hopefully PR won't become a terrible mess. Can't do the map otherwise.

**Comment by SirRichardFrancis** [comment]
Whatever the reason, I just can't update this branch. Going to make new one, with up-to-date build, and redo my changes there.

autism

**Pull request closed without merging**